config/default/manager_config_patch.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: manager
```

Comment:
Should we also change some of these names to be more specific? `codeflare-manager` or something?

Reply:
All the resource names are prefixed using `namePrefix: codeflare-operator-`, as per:

https://github.com/project-codeflare/codeflare-operator/blob/d03836b06599da328684e53cd6ba071eeb6b3e0e/config/default/kustomization.yaml#L9

so the name rendered by Kustomize here is `codeflare-operator-manager`.

Reply:
Ah, missed this. Thanks

Reply:
this /lgtm then

Reply:
this /lgtm then

Reply:
Thanks